### PR TITLE
fix stroke animation for issimple menu

### LINKED ISSUE #64: Make header for Pattern Library mobile ready
Only the o-header is mobile ready. Also the main header of the styleguide should work on mobile

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #64

### CODE REVIEW

**Comment by AndyOGo** [comment]
@LucaMele 
I fixed the `context` bug, it was a hard one...
And I started adding some nice functional utils for debugging...

**Comment by AndyOGo** [comment]
@LucaMele 
Wait we should double-check if that is really fixed (I would really want written tests here...)
Or may this is related to #101 
mobile-navi-deja-vu

File: src/components/m-dropdown/mixins/dropdown.scss [hunk 1]
```diff
@@ -156,7 +156,7 @@
 }
 
 @mixin dropdown-icon($size: $dropdown__icon-size, $right: $dropdown__frame-x-padding, $color: $dropdown__icon-color) {
-  @include hardware-accelerate();
+  // @include hardware-accelerate(); Chrome has an issue
```
Comment (AndyOGo) [inline src/components/m-dropdown/mixins/dropdown.scss]: I really want them for mobile, though I'm fine commenting them out right now

Comment (LucaMele) [reply src/components/m-dropdown/mixins/dropdown.scss]: agree. But curretnly works for mobile as well so its okish. :(

Comment (LucaMele) [reply src/components/m-dropdown/mixins/dropdown.scss]: so can i merge?

Comment (AndyOGo) [reply src/components/m-dropdown/mixins/dropdown.scss]: sure merge it, but I'm really eager and considered to find a cross-browser solution here...

Comment (LucaMele) [reply src/components/m-dropdown/mixins/dropdown.scss]: :) ok
just keep in mind that css 3d transform is also still an experimental feature and could bring other problems in future. I have used it as well lots of time and like the idea of using the gpu, but please consider this as not high prio.

May be worst reading this article here: https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/

Comment (LucaMele) [reply src/components/m-dropdown/mixins/dropdown.scss]: same as WC at the end ;)

**Pull request merged**